util/config/appdir.go

```diff
@@ -84,7 +84,8 @@ func AppDir() string {
 // DownloadsDir returns the absolute path to the Downloads folder in the home folder.
 func DownloadsDir() (string, error) {
 	if runtime.GOOS == "android" {
-		return "", errp.New("android not yet supported")
+		// Android apps are sandboxed, we don't need to specify a folder.
+		return "", nil
```

Comment:
a bit confusing to use DownloadsDir() on Android if it's not the actual DownloadsDir(). not sure if doing sth else here is worth it though

Reply:
Renamed to ExportsDir()